### PR TITLE
KFSPTS-8512 updated log message from debug to info so standard log in…

### PR DESCRIPTION
**Pull request by jhulslander**
…cludes some basic information

### CODE REVIEW

**Comment by jhulslander** [comment]
I added a summary log statement.  Below is an example log file for processing to PDP.  This is what gets sent out if there are no errors.  Do you think more logging would be good in the no error condition?
`2017-04-25 08:36:06,506 [scheduler_Worker-1] INFO  org.kuali.kfs.sys.batch.Job :: Executing job: JobDetail 'unscheduled.concurStandardAccountingExtractToPdpAndCollectorBatchJob':  jobClass: 'org.kuali.kfs.sys.batch.Job isStateful: true isVolatile: false isDurable: true requestsRecovers: false on machine OIT-JDH34-ML scheduler instance id OIT-JDH34-ML1493123284688
{status=Scheduled}
2017-04-25 08:36:06,506 [scheduler_Worker-1] INFO  org.kuali.kfs.sys.batch.Job :: Started processing step: 1=concurStandardAccountingExtractToPdpAndCollectorStep for user <unknown>
2017-04-25 08:36:06,834 [scheduler_Worker-1] INFO  org.kuali.kfs.sys.batch.Job :: Creating user session for step: concurStandardAccountingExtractToPdpAndCollectorStep=kfs
2017-04-25 08:36:08,636 [scheduler_Worker-1] INFO  org.kuali.kfs.sys.batch.Job :: Executing step: concurStandardAccountingExtractToPdpAndCollectorStep=class edu.cornell.kfs.concur.batch.ConcurStandardAccountingExtractToPdpAndCollectorStep
2017-04-25 08:36:08,637 [scheduler_Worker-1] INFO  edu.cornell.kfs.concur.batch.service.impl.ConcurStandardAccountingExtractServiceImpl :: buildListOfFileNamesToBeProcessed number of files found to process: 1
2017-04-25 08:36:08,639 [scheduler_Worker-1] INFO  edu.cornell.kfs.concur.batch.ConcurStandardAccountingExtractToPdpAndCollectorStep :: processCurrentFileAndExtractPdpFeedAndCollectorFromSAEFile, current File: /infra/work/staging/concur/standardAccountingExtract/extract_CES_SAE_v3_20170422081955.txt
2017-04-25 08:36:08,639 [scheduler_Worker-1] INFO  edu.cornell.kfs.concur.batch.service.impl.ConcurStandardAccountingExtractServiceImpl :: parseStandardAccoutingExtractFile, Attempting to parse the file /infra/work/staging/concur/standardAccountingExtract/extract_CES_SAE_v3_20170422081955.txt
2017-04-25 08:36:08,824 [scheduler_Worker-1] INFO  edu.cornell.kfs.concur.batch.service.impl.ConcurStandardAccountingExtractValidationServiceImpl :: validateConcurStandardAccountExtractFile, passed file level validation, the record counts, batch date, and journal totals are all correct.
2017-04-25 08:36:17,258 [scheduler_Worker-1] INFO  edu.cornell.kfs.concur.batch.service.impl.ConcurStandardAccountingExtractServiceImpl :: logPdpSummaryInformation, PDP creation date: 04/22/2017, total number of payments in PDP file: 1, total amount to be paid: 172.50
2017-04-25 08:36:17,276 [scheduler_Worker-1] INFO  edu.cornell.kfs.concur.batch.service.impl.ConcurStandardAccountingExtractServiceImpl :: marshalPdpFeedFile, marshaled the file /infra/work/staging/pdp/paymentImport/pdp_concur_extract_CES_SAE_v3_20170422081955.xml
`

**Comment by db483** [comment]
Hi Jay,

Is there going to be more info in the generated report? If we need to investigate what went wrong I think having a bit more info would be good.( I am thinking about something not working quite as expected although not necessarily being an error. ) Otherwise we would need to always run the file locally to get the debug level info. You are more intimately familiar with the code and the processing so I will defer that judgement to you.

Thank you.

**Comment by jhulslander** [comment]
The report has additional sums and record counts.  To be honest I am not sure what would be helpful or not.  I suppose safer to add more info logging at this point, let me go through some other stuff.

**Comment by jhulslander** [comment]
OK, I suspect most of the concerns will be around why we removed elements from the PDP upload file.  I opened that logging to info.

File: src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountExtractPdpEntryServiceImpl.java [hunk 1]
```diff
@@ -207,25 +207,26 @@ private void addOnlyPositiveAccountingEntryForPDPProcessing(PdpFeedDetailEntry n
...
             newGroupEntry.getDetail().add(newDetailEntry);
         } else {
-            if (LOG.isDebugEnabled()) {
-                StringBuilder sb = new StringBuilder("addNewPaymentDetailToGroupIfTotalIsPositive SAE transactions summed to");
+            if (LOG.isInfoEnabled()) {
+                StringBuilder sb = new StringBuilder("addNewPaymentDetailToGroupIfTotalIsPositive, Not adding detail to group for source document  ");
+                sb.append(newDetailEntry.getSourceDocNbr()).append(" with a tansaction total of ").append(originalDetailTransactionTotal);
                 if (originalDetailTransactionTotal.isNegative()) {
-                    sb.append(" a negative amount in PDP, so it will be handled by collector.");
+                    sb.append(".  A negative amount, it will be handled by the collector.");
```
Comment (db483) [inline src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountExtractPdpEntryServiceImpl.java]: is the extra full stop needed at the beginning of this String message?

File: src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountExtractPdpEntryServiceImpl.java [hunk 2]
```diff
@@ -207,25 +207,26 @@ private void addOnlyPositiveAccountingEntryForPDPProcessing(PdpFeedDetailEntry n
...
+            LOG.info("addOnlyPositiveAccountingEntryForPDPProcessing, not adding accounting entry: " + originalAccountingEntry.toString());
         }
     }
 
     private void addNewPaymentDetailToGroupIfTotalIsPositive(PdpFeedGroupEntry newGroupEntry, PdpFeedDetailEntry newDetailEntry, KualiDecimal originalDetailTransactionTotal) {
-        if (LOG.isDebugEnabled()) {
-            LOG.debug("addNewPaymentDetailToGroupIfTotalIsPositive, total transaction for " + newDetailEntry.getSourceDocNbr() + " detail: " + originalDetailTransactionTotal);
-        }
         if (originalDetailTransactionTotal.isPositive()) {
+            if (LOG.isDebugEnabled()) {
```
Comment (db483) [inline src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountExtractPdpEntryServiceImpl.java]: was this intended to be at debug level?

Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountExtractPdpEntryServiceImpl.java]: That was intentional.  It was intentional, but I am hemming and hawing on it.  Erroring on the side of more logging is better at first, I will switch it.

**Comment by jhulslander** [comment]
made adjustments as requested

**Pull request merged**